### PR TITLE
#159392392 Implement Requests Page Left Sidebar

### PR DESCRIPTION
**Pull request by johnmutuma5**
#### What does this PR do?
Creates a left side bar for navigation

#### Description of Task to be completed?
- Create a left side bar with two links:
    - One link is a dropdown 
- Add icons to the links
- Style the side bar 
- Adds accordion effect to dropdowns
- Adds `enzyme-to-json` snapshot serializer for more readable snapshots

#### How should this be manually tested?
- Clone the repo
- Change into the project directory
- Checkout to the branch `ft-requests-left-bar-#159392392`
- Run the application  
  - `./docker.sh development` for development
  - `./docker.sh production` for production
- Visit `http://localhost:3000/requests`

#### What are the relevant pivotal tracker stories?
#159392392

#### Screenshots (if appropriate)
aug-07-2018 19-50-21

### CODE REVIEW

**Comment by Collins33** [comment]
I like how you have added the gifs to show how your feature works and also the use .
A few things:
- You could update your PR to show how to setup using docker
- Fix the merge conflicts

**Comment by johnmutuma5** [comment]
@amarachukwu-agbo, reviews very much appreciated. Looking into the issues.

File: src/components/LeftSideBar/LeftSideBar.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+class LeftSideBar extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeNavItem: {}
+    };
+  }
+
+  setActiveNavItem = (clickedNavItem) => {
```
[the remark this answers is not in the document]
Comment (samuelkubai) [reply src/components/LeftSideBar/LeftSideBar.jsx]: Thanks @amarachukwu-agbo, on that could we use arrow functions unless we require the normal function implicitly.

[the remark this answers is not in the document]
Comment (johnmutuma5) [reply src/components/LeftSideBar/LeftSideBar.jsx]: @samuelkubai, @amarachukwu-agbo  I think arrow functions are going to be easier to work with when `context` of `this` keyword is a factor.

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["react-app"],
+  "plugins": ["transform-object-rest-spread"]
```
[the remark this answers is not in the document]
Comment (johnmutuma5) [reply .babelrc]: This actually worked well this time, while it was failing at the beginning. Thanks @amarachukwu-agbo for the point.

**Pull request merged**